pkgpanda/docker/dcos-builder/Dockerfile

```diff
@@ -37,12 +37,18 @@ RUN apt-get -qq update && apt-get -y install \
   python-setuptools \
   ruby \
   scala \
+  software-properties-common \
   unzip \
   wget \
   xutils-dev \
   xz-utils \
   zlib1g-dev
 
+# Install glide
+RUN add-apt-repository ppa:masterminds/glide && \
+  apt-get update && \
+  apt-get install -y glide
```

Comment:
Some other packages (e.g. adminrouter) use their own special dcos-builders instead of modifying the default dcos-builder. Or you could do this as part of your `build` script.